convert/convert.go

```diff
@@ -222,7 +222,7 @@ func ThreeWayFaToVcf(f []fasta.Fasta, chr string, out *fileio.EasyWriter) {
 					{
 						Alleles:    []int16{1, 0},
 						Phase:      []bool{false, false},
-						FormatData: []string{"1/0"},
```

Comment:
This string is created from looping through the slice of alleles instead of FormatData, see below:
https://github.com/vertgenlab/gonomics/blob/362af0f127450e9a173350c395594bd7056dd66d/vcf/methods.go#L55-L61